src/index.html

```diff
@@ -11,12 +11,104 @@
       content="ie=edge"
     />
     <title>Moyo header</title>
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500&display=swap"
+      rel="stylesheet"
+    />
     <link
       rel="stylesheet"
       href="./style.css"
     />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header>
+      <a
+        href="#"
+        class="logo"
+      >
+        <img
+          src="images/moyo_1.png"
+          alt="MOYO Logo"
+        />
+      </a>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a
+              href="#"
+              class="nav__link is-active"
+              alt="Apple"
```

Comment:
The `alt` attribute is used here on an anchor tag, which is not correct. The `alt` attribute is meant for `<img>` tags to provide alternative text for images. Consider removing the `alt` attribute from this anchor tag.

Reply:
how i can re-request ?